pyccel/commands/pyccel_clean.py

```diff
@@ -0,0 +1,39 @@
+# coding: utf-8
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+""" Module containing scripts to remove pyccel generated objects
+"""
+import os
+import shutil
+import sysconfig
+
+ext_suffix = sysconfig.get_config_var('EXT_SUFFIX')
+
+def pyccel_clean(path_dir = None, recursive = True, remove_shared_libs = False):
```

Comment:
If used from the command line, I think that we need to provide a proper parser to this small function, or we might even want to use it as a subcommand of `pyccel`. What do you think? Maybe in another PR?

Reply:
Would this prevent me from calling the function from python?

Reply:
I am reusing this file for the teardown. I'm not sure what the effect of calling `parser.parse_args()` repeatedly inside pytest might be

Reply:
I am suggesting to have a separate function that has a parser, and calls this one.

Reply:
Fixed 0e14e63